tools/yulPhaser/main.cpp

```diff
@@ -45,46 +45,12 @@ int main(int argc, char** argv)
 
 		std::cerr << std::endl;
 		std::cerr << "ERROR: " << exception.what() << std::endl;
-		return 1;
+		return 2;
 	}
 	catch (solidity::util::Exception const& exception)
```

Comment:
It has to be `(...)` to catch cover all three kinds of exceptions:
```suggestion
	catch (...)
```

Reply:
It's also failing the Windows build:
```
C:\Users\circleci\project\tools\yulPhaser\main.cpp(50,42): error C2220: the following warning is treated as an error [C:\Users\circleci\project\build\tools\yul-phaser.vcxproj]
  Building Custom Rule C:/Users/circleci/project/solc/CMakeLists.txt
C:\Users\circleci\project\tools\yulPhaser\main.cpp(50,42): warning C4101: 'exception': unreferenced local variable [C:\Users\circleci\project\build\tools\yul-phaser.vcxproj]
```